### ISSUE TITLE
Locking key as in QMK

### ISSUE THREAD

**Issue by fenuks** [kmonad/kmonad#470]
I have one keyboard with QMK, and would like to replicate its functionality with KMonad. I went through tutorial, and browsed documentation, but it seems locking functionality is not supported.

It works this way, QMK specifies `KC_LOCK` key, and upon pressing it, *next* pressed key will be locked. Lock is released when this locked key is pressed again. This is very useful, because it allows to have dynamical sticky keys.without time limit. 

https://github.com/qmk/qmk_firmware/blob/master/docs/keycodes.md#key-lock-idkey-lock

**Comment by slotThe** [comment]
Indeed, not implemented right now; marking this as a feature request
